odoo/models.py

```diff
@@ -5123,6 +5123,25 @@ def copy(self, default=None):
 
         return record_copy
 
+    @api.returns(None, lambda value: value.ids)
```

Comment:
Decorator is incorrect.
```suggestion
    @api.returns('self')
```